.github/workflows/release.yaml

```diff
@@ -100,18 +100,19 @@ jobs:
         uses: actions/checkout@v3
         with:
           ref: docs
-        
+          fetch-depth: 0
```

Comment:
I appreciate this is not an optimal approach could become very slow, however I have tried a few different fetch configurations without success (in the CI). I would like to use this configuration for now to unblock the workflow and investigate in isolation.